templates/layouts/aw-install-simg.hbs

```diff
@@ -15,58 +15,85 @@
       <p>Before installing AsteroidOS to your watch, make sure you are aware of the capabilities and limitations of AsteroidOS on the {{title}}. The following table should summarize the current support of this watch:</p>
       {{> body }}
       <h1>Preparation</h1>
-      <div class="install-preparation-box">
-          <img src="{{assets}}/img/{{deviceName}}.jpg" class="install-preparation-img"><br>
-      </div>
       <div class="install-preparation-box">
         <h3>Download AsteroidOS 1.0</h3>
         <a class="btn btn-primary" href="https://release.asteroidos.org/1.0/{{deviceName}}/asteroid-image-{{deviceName}}.ext4" role="button">asteroid-image-{{deviceName}}.ext4</a>
         <a class="btn btn-primary" href="https://release.asteroidos.org/1.0/{{deviceName}}/zImage-dtb-{{deviceName}}.fastboot" role="button">zImage-dtb-{{deviceName}}.fastboot</a>
-        <h3>Install Fastboot & Adb</h3>
+        <br><br><i>Place the files in your users "Downloads" folder for the later commands to work</i><br>
+        <h3>Install Fastboot & ADB</h3>
+        <b>Linux systems</b> with apt package manager
```

Comment:
Here you say "Linux systems" and then you say "Windows users". Which isn't really of the same "type".
Maybe you should say "On Linux systems" and "On Windows systems". Or "For Linux users" and "For Windows users"

Reply:
systems is the way to go.